example/files-info/main.go

```diff
@@ -0,0 +1,113 @@
+package main
```

Comment:
Can I see the code for this server on Github?

Reply:
https://github.com/savel999/graphql-file-info